src/PKSim.Core/PKSimColors.cs

```diff
@@ -21,20 +21,20 @@ public static class PKSimColors
       /// </summary>
       public static Color ChartBack
       {
-         get { return Colors.ChartBack; }
-         set { Colors.ChartBack = value; }
+         get => Colors.ChartBack;
+         set => Colors.ChartBack = value;
       }
 
       public static Color ChartDiagramBack
       {
-         get { return Colors.ChartDiagramBack; }
-         set { Colors.ChartDiagramBack = value; }
+         get => Colors.ChartDiagramBack;
+         set => Colors.ChartDiagramBack = value;
       }
 
       /// <summary>
       /// Color used for cell that are locked/disabled 
       /// </summary>
-      public static Color Disabled = Colors.Disabled;
+      public static Color Disabled = Color.FromArgb(255, 247, 247, 249);
```

Comment:
This is the default color that looks good for the new stuff. If we agree that we like it, I'll move it to core